src/coreComponents/physicsSolvers/wavePropagation/AcousticWaveEquationSEM.cpp

```diff
@@ -163,6 +134,33 @@ void AcousticWaveEquationSEM::registerDataOnMesh( Group & meshBodies )
 }
 
 
+void AcousticWaveEquationSEM::postProcessInput()
+{
+
+  GEOSX_THROW_IF( m_sourceCoordinates.size( 1 ) != 3,
+                  "Invalid number of physical coordinates for the sources",
+                  InputError );
+
+  GEOSX_THROW_IF( m_receiverCoordinates.size( 1 ) != 3,
+                  "Invalid number of physical coordinates for the receivers",
+                  InputError );
+
+  localIndex const numNodesPerElem = 8;
+
+  localIndex const numSourcesGlobal = m_sourceCoordinates.size( 0 );
+  m_sourceNodeIds.resizeDimension< 0, 1 >( numSourcesGlobal, numNodesPerElem );
+  m_sourceConstants.resizeDimension< 0, 1 >( numSourcesGlobal, numNodesPerElem );
```

Comment:
These are 2D arrays right? In that case you can just do `m_sourceNodeIds.resize( numSourcesGlobal, numNodesPerElem )` and call it good.